packages/pn-personagiuridica-webapp/src/redux/apikeys/reducers.ts

```diff
@@ -0,0 +1,68 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+import {
+  CheckIssuerStatus,
+  PublicKeyBaseParams,
+  PublicKeys,
+  VirtualKeyBaseParams,
+  VirtualKeys,
+} from '../../models/ApiKeys';
+import {
+  checkPublicKeyIssuer,
+  createPublicKey,
+  createVirtualApiKey,
+  getPublicKeys,
+  getVirtualApiKeys,
+} from './actions';
+
+type initialStateType = {
+  loading: boolean;
+  publicKeys: PublicKeys;
+  virtualKeys: VirtualKeys;
+  publicKey: PublicKeyBaseParams | null;
+  virtualKey: VirtualKeyBaseParams | null;
+  issuerStatus: CheckIssuerStatus | null;
+};
+
+const initialState: initialStateType = {
+  loading: false,
+  publicKeys: {
+    items: [],
+    total: 0,
+  },
+  virtualKeys: {
+    items: [],
+    total: 0,
+  },
+  publicKey: null,
```

Comment:
Do we need to store publicKey and virtualKey to create an api key?

Reply:
removed